src/main/java/minicraft/core/Renderer.java

```diff
@@ -78,7 +80,36 @@ private Renderer() {}
 		return new SpriteSheet[] { itemSheet, tileSheet, entitySheet, guiSheet, skinsSheet };
 	}
 
-	static void initScreen() {
+	public static void setAspectRatio() {
+		int width = 0;
+		int height = 0;
+
+		switch ((String) Settings.get("aspectratio")) {
+			case "Native":
+				java.awt.Dimension size = java.awt.Toolkit.getDefaultToolkit().getScreenSize();
+				width = (int) size.getWidth();
+				height = (int) size.getHeight();
+				break;
+			
+			case "4x3":
+				width = 4;
+				height = 3;
+				break;
+
+			case "16x9":
+				width = 16;
+				height = 9;
+				break;
+			default:
+				break;
+		}
+
+		double s = Math.min((double) Renderer.MAX_SIZE/width, (double) Renderer.MAX_SIZE/height);
+		Renderer.HEIGHT = (int) (height * s);
```

Comment:
I can't think how this would work in my head alone. Does this secure that the scale of the window is the same as before?